xep-0369.xml

```diff
@@ -637,9 +637,6 @@
     to='mix.shakespeare.example'
     from='hag66@shakespeare.example/pda'
     id='7nve413p'>
-  <register xmlns='urn:xmpp:mix:0'>
```

Comment:
I feel that including the original payload is rather useless and we shouldn't encourage this behavior, so either way I think it's worth removing this.